code/__HELPERS/icons.dm

```diff
@@ -1370,3 +1370,26 @@ GLOBAL_LIST_EMPTY(transformation_animation_objects)
 	copy.underlays = underlays_to_keep
 
 	return copy
+
+/// Returns the (isolated) security HUD icon for the given job.
+/proc/get_job_hud_icon(datum/job/job) as /icon
+	RETURN_TYPE(/icon)
```

Comment:
```suggestion
```
not needed anymore (i wrote the original code before https://github.com/SpaceManiac/SpacemanDMM/commit/86172de96511aa8d10508fb3bf6a6df36bf9d311)